apps/api/v2/src/modules/tokens/tokens.repository.ts

```diff
@@ -52,7 +55,7 @@ export class TokensRepository {
       try {
         await this.dbWrite.prisma.$transaction([
           this.dbWrite.prisma.accessToken.deleteMany({
-            where: { client: { id: clientId }, userId: ownerId, expiresAt: { lte: new Date() } },
```

Comment:
Removed `expiresAt: { lte: new Date() }` because we don't have it for refresh tokens and with it for some reason tests locally resulted in a conflict of trying to create an access token with already existing secret 🤷 it serves no purpose and we can delete all user access tokens before creating new one.